core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3873,12 +3886,16 @@ public synchronized HttpRedirect doQuietDown() {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
+     * @param reason Quiet reason that will be visible to user
      */
     @RequirePOST
-    public HttpRedirect doQuietDown(@QueryParameter boolean block, @QueryParameter int timeout) throws InterruptedException, IOException {
+    public HttpRedirect doQuietDown(@QueryParameter boolean block,
```

Comment:
Breaking change of non-`@Restricted` API. At a minimum, confirm no uses in the plugin ecosystem.

Reply:
Added a method with an old signature for compatibility